Rocket.Chat/Managers/AppManager.swift

```diff
@@ -164,7 +163,7 @@ extension AppManager {
         func openRoom() -> Bool {
             guard let channel = Subscription.find(name: name, subscriptionType: [type]) else { return false }
 
-            ChatViewController.shared?.subscription = channel
+            // ChatViewController.shared?.subscription = channel
```

Comment:
Regression: will not open channel or group after pressing notification

Reply:
✅